src/app/components/transcriptions/transcriptions.component.html

```diff
@@ -12,81 +12,109 @@
+      <h2 class="govuk-heading-m">In Progress</h2>
+      <app-data-table
+        id="in-progress-table"
+        [rows]="vm.inProgessRequests"
+        [columns]="columns"
+        [pagination]="true"
+        [pageLimit]="15"
+      >
+        <ng-template [tableRowTemplate]="vm.inProgessRequests" let-row>
+          <ng-container *ngTemplateOutlet="tableTemplatePartial; context: { $implicit: row }"></ng-container>
+        </ng-template>
+      </app-data-table>
+    }
+
+    @if (vm.completedRequests.length) {
+      <h2 class="govuk-heading-m">Ready</h2>
+      <div class="action-container">
+        <div class="govuk-hint">Select to apply actions</div>
+        <button id="delete-button" class="govuk-button govuk-button--secondary">Remove transcript request</button>
+      </div>
+      <app-data-table
+        id="ready-table"
+        [rows]="vm.completedRequests"
+        [columns]="readyColumns"
+        [pagination]="true"
+        [pageLimit]="15"
+        [rowSelectable]="true"
+      >
+        <ng-template [tableRowTemplate]="vm.completedRequests" let-row>
+          <ng-container *ngTemplateOutlet="tableTemplatePartial; context: { $implicit: row }"></ng-container>
+          <td class="govuk-table__cell">
+            <a [routerLink]="'.'">View</a>
+          </td>
+        </ng-template>
+      </app-data-table>
+    }
+  } @else {
+    <app-loading text="Loading transcription requests..." />
+  }
+</ng-template>
+
+<ng-template #ApproverTemplate>
+  @if (approverRequests$ | async; as approverRequests) {
+    @if (!approverRequests.length) {
+      <p id="no-requests-message" class="govuk-body">There are no transcription requests to review</p>
+    }
+    @if (approverRequests.length) {
+      <h2 class="govuk-heading-m">Requests to approve or reject</h2>
+
+      <app-data-table
+        id="approver-table"
+        [rows]="approverRequests"
+        [columns]="approverColumns"
+        [pagination]="true"
+        [pageLimit]="15"
+      >
+        <ng-template [tableRowTemplate]="approverRequests" let-row>
+          <td class="govuk-table__cell">{{ row.case_number }}</td>
+          <td class="govuk-table__cell">{{ row.courthouse_name }}</td>
+          <td class="govuk-table__cell">{{ row.hearing_date | date: 'dd MMM yyyy' }}</td>
+          <td class="govuk-table__cell">{{ row.transcription_type }}</td>
+          <td class="govuk-table__cell">{{ row.requested_ts | date: 'dd MMM yyyy HH:mm' }}</td>
+          <td class="govuk-table__cell">{{ row.transcription_id }}</td>
+          <td class="govuk-table__cell">
+            <span [ngClass]="row.urgency === 'Overnight' ? 'moj-badge moj-badge--red' : ''">{{ row.urgency }}</span>
+          </td>
+          <td class="govuk-table__cell">
+            <a [routerLink]="'.'">View</a>
+          </td>
+        </ng-template>
+      </app-data-table>
+    }
+  } @else {
+    <app-loading text="Loading transcriptions awaiting approval..." />
+  }
+</ng-template>
```

Comment:
This template file is pretty big, maybe a candidate to split up into separate components at some point?